src/main/resources/hudson/plugins/sauce_ondemand/credentials/SauceCredentials/credentials.jelly

```diff
@@ -10,9 +10,9 @@
         <select name="restEndpoint">
             <option value="https://saucelabs.com/" selected="${instance.restEndpoint.equals('https://saucelabs.com/') ? 'true' : null}">US</option>
             <option value="https://eu-central-1.saucelabs.com/" selected="${instance.restEndpoint.equals('https://eu-central-1.saucelabs.com/') ? 'true' : null}">EU</option>
-            <option value="https://us-east-1.saucelabs.com/" selected="${instance.restEndpoint.equals('https://us-east-1.saucelabs.com/') ? 'true' : null}">US_EAST</option>
+            <option value="https://us-east-1.saucelabs.com/" selected="${instance.restEndpoint.equals('https://us-east-1.saucelabs.com/') ? 'true' : null}">US_EAST 1 (deprecated)</option>
```

Comment:
I wonder if deprecated notice fits it well. We might support it in the future, right?

Reply:
This is referring to the old US East environment so I don't think we'll support it in the future. But it's not really in scope for the `sc legacy` changes, so I've removed it from the PR.